docs/API.md

```diff
@@ -78,11 +90,10 @@ function myApplicationLogic (argOne, activeSpan) {
 }
 ```
 
-```javascript
-// automatic context propagation via the scope manager
+Automatically propagate context with the scope manager after you initialize
+the tracer:
```

Comment:
This is where the above statement is relevant: https://github.com/signalfx/signalfx-nodejs-tracing/pull/57/files#r425247772